From 74af93f5298b54bc2327843f390bf202776f2f48 Mon Sep 17 00:00:00 2001
From: Oswald Buddenhagen <oswald.buddenhagen@qt.io>
Date: Tue, 10 Jan 2017 14:18:02 +0100
Subject: [PATCH] fix parallel builds with -qt-freetype -system-libpng

freetype has no dependency on gui, so it needs to pull in gui's
configuration manually, as that's where the system libpng is found.

Task-number: QTBUG-58038
Change-Id: I881495f7d2a8f7c1a45d7d4c9e7698ff1d30f2a9
Reviewed-by: Joerg Bornemann <joerg.bornemann@qt.io>
Reviewed-by: Joni Poikelin <joni.poikelin@qt.io>

Upstream: https://code.qt.io/cgit/qt/qtbase.git/commit/?id=4dcfd90e4fd7d4c49138038dbbcbda8794a9fbff
Signed-off-by: Peter Seiderer <ps.report@gmx.net>
---
 src/3rdparty/freetype/freetype.pro | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/3rdparty/freetype/freetype.pro b/src/3rdparty/freetype/freetype.pro
index 5b1eb92e..390a6da7 100644
--- a/src/3rdparty/freetype/freetype.pro
+++ b/src/3rdparty/freetype/freetype.pro
@@ -69,6 +69,7 @@ DEFINES += FT_CONFIG_OPTION_SYSTEM_ZLIB
 include(../zlib_dependency.pri)
 
 DEFINES += FT_CONFIG_OPTION_USE_PNG
+include($$OUT_PWD/../../gui/qtgui-config.pri)
 QMAKE_USE_PRIVATE += libpng
 
 DEFINES += TT_CONFIG_OPTION_SUBPIXEL_HINTING
-- 
2.11.0

